src/style.css

```
@@ -0,0 +1,55 @@
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo__link {
+  margin: 8px 0 8px 50px;
+
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  list-style-type: none;
+}
+
+.nav__link {
+  padding: 22.5px 0;
+  margin: 0 10px;
+  font-family: "Roboto", sans-serif;

```

```suggestion
  font-family: "Roboto", sans-serif;
```
font-family should be in the `html` selector